geode-core/src/main/java/org/apache/geode/internal/cache/backup/BackupFileCopier.java

```diff
@@ -167,7 +168,13 @@ private void copyOplogFile(DiskStore diskStore, DirectoryHolder dirHolder, File
 
     Path tempDiskDir = temporaryFiles.getDiskStoreDirectory(diskStore, dirHolder);
     try {
-      createLink(tempDiskDir.resolve(file.getName()), file.toPath());
+      if (SystemUtils.isWindows()) {
+        // Hard links cannot be deleted on Windows if the process is still running, so prefer to
+        // actually copy the files.
+        FileUtils.copyFileToDirectory(file, tempDiskDir.toFile());
+      } else {
+        createLink(tempDiskDir.resolve(file.getName()), file.toPath());
+      }
```

Comment:
It looks like there is fallback logic just below your change to try copying the file if hard-linking failed. Maybe that should be moved into your `else` branch now?